### ISSUE TITLE
TT Infused crops compat not working

### ISSUE THREAD

**Issue by Zero-ui9** [ReikaKalseki/Reika_Mods_Issues#2837]
http://puu.sh/HDICG/358d238cb3.jpg

The infused crops aren't being harvested, despite there being support for them in DragonAPI. I don't have RoC installed, so I can only demonstrate with the elemental harvester(also the tile accelerators are there to speed up their growth because they don't work off normal growth ticks, but rather the chance to grow is calculated in the tile entity).

**Comment by ReikaKalseki** [comment]
What is in your log regarding loading that crop type?

**Comment by Zero-ui9** [comment]
http://puu.sh/HDKLm/68ee33b02e.png 
basically just logging an oredict entry for them. unless a log is missing that is expected to be there, the chat logs are from a portable scanner

**Comment by ReikaKalseki** [comment]
No, you should have a section somewhere when ModCropList is initialized.

**Comment by Zero-ui9** [comment]
I cannot find any mention of "ModCropList" at all, nor anything else indicating an error related to this in my logs.

**Comment by ReikaKalseki** [comment]
Look for `Error loading crop`.
